DeepCat.js

```diff
@@ -495,45 +500,40 @@
 	 * @param {string} str
 	 * @return {string}
 	 */
-	function makeHintboxCookieToken( str ) {
+	function makeHintboxSettingToken( str ) {
 		return String( djb2Code( str ) );
 	}
 
-	function hasHintCookie() {
+	function hasHintSetting() {
```

Comment:
shouldHideHint is a bit tricky due to the global shouldHideHints named it checkShouldHideHints now